plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -354,6 +354,60 @@ var _ = Describe("validator", func() {
 			admissionHandler.SetAuthorizer(auth)
 		})
 
+		Context("cluster autoscaler configuration check for ScaleDownUnneededTime and ScaleDownDelayAfterAdd time", func() {
+			BeforeEach(func() {
+				shoot.Spec.Kubernetes.ClusterAutoscaler = &core.ClusterAutoscaler{}
+			})
+
+			DescribeTable("validate cluster autoscaler configuration on shoot create", func(workerUnneededTime, unneededTime, afterAddTime *metav1.Duration, errorMatcher types.GomegaMatcher) {
+				var (
+					worker core.Worker
+				)
+
+				shoot.Spec.Kubernetes.ClusterAutoscaler = &core.ClusterAutoscaler{
+					ScaleDownDelayAfterAdd: afterAddTime,
+					ScaleDownUnneededTime:  unneededTime,
+				}
+
+				worker = core.Worker{
+					Name: "machine-name-1",
+					Machine: core.Machine{
+						Type: "machine-type-1",
+						Image: &core.ShootMachineImage{
+							Name: validMachineImageName,
+						},
+						Architecture: ptr.To("amd64"),
+					},
+					Minimum: 1,
+					Maximum: 1,
+					ClusterAutoscaler: &core.ClusterAutoscalerOptions{
+						ScaleDownUnneededTime: workerUnneededTime,
+					},
+				}
+
+				shoot.Spec.Provider.Workers = append(shoot.Spec.Provider.Workers, worker)
```

Comment:
```suggestion				
                                worker2 := core.Worker{
					Name: "machine-name-2",
					Machine: core.Machine{
						Type: "machine-type-1",
						Image: &core.ShootMachineImage{
							Name: validMachineImageName,
						},
						Architecture: ptr.To("amd64"),
					},
					Minimum: 1,
					Maximum: 1,
				}

				shoot.Spec.Provider.Workers = append(shoot.Spec.Provider.Workers, worker, worker2)
```
Given this the scenario `should allow when configuration for worker ScaleDownUnneededTime is > 1m, ScaleDownUnneededTime is < 1m and ScaleDownDelayAfterAdd is < 1m` should fail but instead it passes